manifests/ops-files/change-cidrs.yml

```diff
@@ -12,9 +12,8 @@
   value: [((kubedns_service_ip))]
 
 - type: replace
-  path: /instance_groups/name=worker/jobs/name=flanneld/properties?
-  value:
-    pod-network-cidr: ((pod_network_cidr))
+  path: /instance_groups/name=worker/jobs/name=flanneld/properties?/pod-network-cidr?
```

Comment:
This is also handled in PR #391 - we can close that one down once this is merged.

Reply:
Ohk, I'll close that one once merge this one